account_invoice_import_ubl/tests/files/UBLKetentest_Referentiefactuur_20150100.xml

```diff
@@ -0,0 +1,242 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
Is there not a more generic file name that can be used? `invoice_1.xml` for instance?